src/alwaysAck.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+/* @flow */
+
+import type { MessageHandler } from './index'
+
+import { ackOnComplete, ackOnError } from './ackOn'
+import parseAndHandleMessage from './parseAndHandleMessage'
+
+export default (handleError:(e:mixed) => Promise<mixed>, f:(message:Object) => Promise<mixed>): MessageHandler =>
+  parseAndHandleMessage(x => x, ackOnError(handleError), ackOnComplete, f)
```

Comment:
> Do you have a suggestion for how to get to a better spot?

That depends on how you define "better spot"!  

In any case, I don't think `alwaysAck` _needs_ to share code with the "parseAndHandle" convenience functions.  It's a different thing and combining them together feels wrong to me because I specifically intended to separate parsing and acking policies.  🐼 

Some suggestions (not mutually exclusive):

1. Investigate whether my previous assertion ("Choosing the acking policy separately from the encoding policy makes more sense") is true by creating a few more policies.
2. Leave `alwaysAck` out of this repo for now
3. Leave `alwaysAck` in the repo (but not sharing code?).

I had a brief thought that we might just want to revisit `alwaysAck` when we look into retry policies.  Does that make sense?  If so, then perhaps, (2) is the right path?